src/Grpc.AspNetCore.Server/Internal/CallHandlers/ServerCallHandlerBase.cs

```diff
@@ -18,6 +18,7 @@
 
 using System;
 using System.Threading.Tasks;
+using Grpc.AspNetCore.Server.Feature;
```

Comment:
Is there a really good reason to create a namespace for this?

Reply:
Nothing other than features everywhere else live in a Features namespace.

I'm going to leave it here for now and we can change it in an API review closer to release.

Reply:
IMO this naming doesn't make sense in our code, but is excusable because we have lots of features. I can't imagine grpc having too many, but maybe I need to adopt a growth mindset!